### PR TITLE
CERD-178 > Drop Down

### PR DESCRIPTION
**Pull request by chrisunl**

### CODE REVIEW

File: packages/drop-down/demo/index.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.option-item:hover, .option-item:focus {
+    background-color: var(--theme-light-blue, lightblue);
```
Comment (drochgenius) [inline packages/drop-down/demo/index.css]: Well, not extactly it.
But ok, keep it like this, we'll have to discuss theming as a common feature for all components.
Leave it like this for now.

File: packages/drop-down/src/components/drop-down.ts [hunk 2]
```diff
@@ -1,62 +1,76 @@
-import { ComponentBase, html, TemplateResult, Feedback, applyMixins } from '@hmh/component-base/dist/index';
-import { MDCSelect } from '@material/select/index.js';
+import { ComponentBase, html, TemplateResult } from '@hmh/component-base/dist/index';
+// @ts-ignore : no type definition available
```
Comment (drochgenius) [inline packages/drop-down/src/components/drop-down.ts]: I think you can remove the `ts-ignore` and add a declaration in the file `typings.d.ts`

File: packages/drop-down/src/components/drop-down.ts [hunk 3]
```diff
@@ -1,62 +1,76 @@
...
+        } else {
+            this.onItemClicked(event, eventTarget.parentNode as HTMLElement)
+        }
+    }
+
+    protected _didRender(): void {
+        this.enableAccessibility();
     }
 
     protected _render({ values }: DropDown): TemplateResult {
```
Comment (drochgenius) [inline packages/drop-down/src/components/drop-down.ts]: Now you need to add `open` in the props object passed in params `{open, values}:DropDown`.
And use `open` prop to manage the state of the drop down list.

File: packages/drop-down/demo/index.css (hunk 1 above)
Comment (chrisunl) [reply packages/drop-down/demo/index.css]: Updated

File: packages/drop-down/src/components/drop-down.ts (hunk 3 above)
Comment (chrisunl) [reply packages/drop-down/src/components/drop-down.ts]: Done

**Pull request merged**